.gitignore

```diff
@@ -99,3 +99,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# clasp configuration file
+.clasp.json
```

Comment:
Normal projects should be committing this right?
I which case I would prefer to have the .gitignore ready to make it easier for the user rather than easier the skeletop development itself.

Reply:
normal project should not